sky/sky_logging.py

```diff
@@ -25,18 +27,66 @@ def format(self, record):
         return msg
 
 
+_root_logger = logging.getLogger('sky')
+_default_handler = None
+_logging_config = threading.local()
+_logging_config.is_silent = False
+
+echo = print
```

Comment:
Is it possible to make it so that `sky_logging.print = print`? Here we should also add a comment:
```
# All code inside the library should use sky_logging.print() rather than print().
# This is to make controlled logging via is_silent() possible: in some situation we would like to disable any printing/logging.
```
The reason of this renaming is to make it easier for reviews to spot accidental usage of `print()` / for authors to remember it more easily.